Userland/Applets/Network/main.cpp

```diff
@@ -134,7 +137,12 @@ class NetworkWidget final : public GUI::ImageWidget {
             if (!adapter_info.is_empty())
                 adapter_info.append('\n');
 
-            adapter_info.appendff("{}: {}", ifname, ip_address);
+            adapter_info.appendff("{}: {} ", ifname, ip_address);
+            if (!link_up)
+                adapter_info.appendff("(down)");
+            else
+                adapter_info.appendff("({} Mb/s, {}-duplex)", link_speed,
+                    link_fd ? "full" : "half");
```

Comment:
I'm not entirely sure if information about whether the connection is full or half-duplex is useful enough to put on a tooltip tbh

Reply:
I removed it